src/components/LaunchDetailsPage/LaunchDetailsPage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  Container,
+  Grid,
+  Card,
+  CardActionArea,
+  CardMedia,
+} from '@mui/material';
+import { SecondaryButton } from '../../shared/components/SecondaryButton';
+import { LaunchCardContent } from './components/LaunchCardContent';
+import { launchDetails } from './mocks/launchDetails';
+
+export const LaunchDetailsPage = () => {
+  return (
+    <Container>
+      <SecondaryButton path="/" caption="go back" />
+      <Card raised>
+        <CardActionArea>
+          <Grid container columns={{ xs: 4, sm: 8, md: 12 }}>
+            <Grid item xs={4} sm={3} md={4}>
+              <CardMedia
+                src={launchDetails.image}
+                alt={launchDetails.name}
+                height="100%"
+                component="img"
+              />
+            </Grid>
+            <Grid item xs={4} sm={5} md={8} sx={{ p: 2 }}>
+              <LaunchCardContent />
+            </Grid>
```

Comment:
1. Is it possible to move grid item to `LaunchCardContent`?
2. why do we need `sx={{ p: 2 }}`?

Reply:
1. I think it doesn't make sense because this `Grid` - the second `Grid item` included to `Grid container`.